utils/regex.ts

```diff
@@ -0,0 +1,33 @@
+import safeRegex from "safe-regex";
+
+/**
+ * 判断一个字符串是否看起来像正则表达式
+ * @param str 待测试字符串
+ */
+export function isLooksLikeRegexString(str: string) {
+  // 1. 基本格式：以斜杠开头、斜杠结尾，可跟 0~6 位正则标志
+	const regexSyntax = /^\/(.+)\/([gimsuy]*)$/;
```

Comment:
```suggestion
	const regexSyntax = /^\/(.+)\/([a-z]*)$/;
```

你永远不能预计未来 tc39 会增加哪些新的正则标志。
就比如你目前给的六位标志就不包含我最喜欢的 v 正则标志。